vscode/package.json

```diff
@@ -153,43 +161,43 @@
             }
         ],
         "menus": {
-            "editor/title": [
```

Comment:
I am slightly worried about moving those controls to the side, since they are not "always on" and one might not know that they are available.

Reply:
Could we move them to the bottom?

Reply:
My view was that since this is the view for codios, this should also be the central area to manipulate them. `editor/title` controls should be used for the specific file displayed. To me a codio is a collection of media and not one specific file. I understand the concern but I believe once shown to a new user it will make sense.

> Could we move them to the bottom?

In a later PR (add_statusbar-ui) I do have a status bar indicator to signify the playing of a codio, if that is what you meant by moving them to the bottom.